packages/hooks/index.d.ts

```diff
@@ -2,8 +2,9 @@ declare interface Window {
   b2b: {
     callbacks: {
       dispatchEvent: (
-        callbackKey: import('./useCustomCallbacks').CallbackKey,
-        data: unknown,
+        callbackKey: import('./useB2BCallback').B2BEvent,
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        data?: any,
```

Comment:
maybe we should enforce the `Record` on the data field with keys being string, and values being anything.

https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

Reply:
I pushed a change, but using with `Record` with any as values is basically the same as having that any. If we want be specific about types we would required to build a proper interface every single type that's being used or might be used in the future and would require way more effort.